packages/meditrak-server/src/permissions/constants.js

```diff
@@ -4,4 +4,5 @@
  */
 
 export const BES_ADMIN_PERMISSION_GROUP = 'BES Admin';
+export const LESMIS_ADMIN_PERMISSION_GROUP = 'Laos Schools Admin';
```

Comment:
Damn, I thought we'd set up the LESMIS permission groups but it doesn't look like that ever happened. This should be `LESMIS Admin`, and we should alter the permission groups in the database to be as we had planned: LESMIS Admin -> LESMIS User -> LESMIS Public